src/FedSrv/FedSrv.CPP

```diff
@@ -9584,6 +9584,11 @@ void LoadSettings()
 		DWORD cbLeaderboardUpdateUrl = sizeof(g.szLeaderboardUpdateUrl);
 		RegQueryValueExA(hKey, "LeaderboardUpdateUrl", NULL, &dwType, (unsigned char*)g.szLeaderboardUpdateUrl, &cbLeaderboardUpdateUrl);
 
+        // Loran - Game statistics logging
+        g.szGameStatsUpdateUrl[0] = '\0';
+        DWORD cbGameStatsUpdateUrl = sizeof(g.szGameStatsUpdateUrl);
+        RegQueryValueExA(hKey, "GameStatsUpdateUrl", NULL, &dwType, (unsigned char*)g.szGameStatsUpdateUrl, &cbGameStatsUpdateUrl);
```

Comment:
Does this work if the registry setting is missing, or will it crash?

Reply:
good catch, it did but latest commit should fix it.